FluentScreenRecorder/MultilingualResources/FluentScreenRecorder.it.xlf

```diff
@@ -104,15 +104,15 @@
         </trans-unit>
         <trans-unit id="RecordingStart" translate="yes" xml:space="preserve">
           <source>Start Recording</source>
-          <target state="translated">Inizia a registrare</target>
+          <target state="translated">Inizia registrazione</target>
```

Comment:
Molto meglio così, anche se metterei l'articolo (inizia la registrazione). Stessa cosa per interrompi la registrazione

Reply:
Di solito in termini tecnici l'articolo viene omesso per dare importanza al contenuto. Il tooltip diventa più corto e l'utente capisce immediatamente il concetto. Comunque sia sostituirò i termini nel prossimo commit.